### PR TITLE
Modify Design Cards

### PR DESCRIPTION
**Pull request by filippoweb3**
Current Cards do not adapt to screen size

- Cards
- Banner

### CODE REVIEW

**Comment by DrW3RK** [comment]
If there is an accessibility issue with the Wiki, we need to sort it out with the web design team. 

image

The elements here are redundant. The "click" text, as well as "videos youtube click" can be removed, which keeps it concise. Also, "built with love" with Polkadot does not go with the formal and technical tone of the Wiki.

The same goes with the redundancy with the the hyperlinks on the image cards and the numerous "view page" hyperlinks is visually unappealing. Why have you chosen to make these changes?

image

**Comment by filippoweb3** [comment]
@DrW3RK I removed the badges and modified the cards (see below)

Screenshot 2024-01-16 at 16 20 45

The title of the cards has the link. If you are ok with it I can transfer those changes to all other pages that are affected.

File: docs/general/governance-apps.md [hunk 1]
```diff
@@ -32,7 +32,18 @@ To make OpenGov multi-role delegation easy and intuitive,
...
-[![Delegation Dashboard Tutorial](https://img.youtube.com/vi/RapBYZc5ZPo/0.jpg)](https://www.youtube.com/watch?v=RapBYZc5ZPo)
+<div className="row">
+  <div className="col text--center">
+    <a href="https://www.youtube.com/watch?v=RapBYZc5ZPo">
+      <img src="https://img.youtube.com/vi/RapBYZc5ZPo/0.jpg" width="350" style={{ borderRadius: 10, border: '1px solid slategrey' }} />
+    </a>
+    <p>
+      Delegation Dashboard Tutorial
+      <br />
+      <a href="https://www.youtube.com/watch?v=RapBYZc5ZPo">» view video</a>
```
Comment (profradha) [inline docs/general/governance-apps.md]:
```suggestion
      <a href="https://www.youtube.com/watch?v=RapBYZc5ZPo">Delegation Dashboard Tutorial</a>
```

File: docs/general/polkadot-vault.md [hunk 2]
```diff
@@ -32,7 +32,18 @@ Vault app.
...
-[![Polkadot Vault Tutorial](https://img.youtube.com/vi/IG_RGLsb2g0/0.jpg)](https://www.youtube.com/watch?v=IG_RGLsb2g0)
+<div className="row">
+  <div className="col text--center">
+    <a href="https://www.youtube.com/watch?v=IG_RGLsb2g0">
+      <img src="https://img.youtube.com/vi/IG_RGLsb2g0/0.jpg" width="350" style={{ borderRadius: 10, border: '1px solid slategrey' }} />
+    </a>
+    <p>
+      How to Use Polkadot Vault
+      <br />
+      <a href="https://www.youtube.com/watch?v=IG_RGLsb2g0">» view video</a>
```
Comment (profradha) [inline docs/general/polkadot-vault.md]:
```suggestion
      <a href="https://www.youtube.com/watch?v=IG_RGLsb2g0"> How to Use Polkadot Vault</a>
```

File: docs/general/scams.md [hunk 3]
```diff
@@ -19,7 +19,18 @@ with anyone, they can access your account, including your funds. This informatio
...
-[![Account Creation Tutorial](https://img.youtube.com/vi/ARsdXZycJAg/0.jpg)](https://www.youtube.com/watch?v=ARsdXZycJAg)
+<div className="row">
+  <div className="col text--center">
+    <a href="https://www.youtube.com/watch?v=ARsdXZycJAg">
+      <img src="https://img.youtube.com/vi/ARsdXZycJAg/0.jpg" width="350" style={{ borderRadius: 10, border: '1px solid slategrey' }} />
+    </a>
+    <p>
+      Stay Safe in Polkadot
+      <br />
+      <a href="https://www.youtube.com/watch?v=ARsdXZycJAg">» view video</a>
```
Comment (profradha) [inline docs/general/scams.md]:
```suggestion
      <a href="https://www.youtube.com/watch?v=ARsdXZycJAg"> Stay Safe in Polkadot</a>
```

File: docs/general/staking-apps.md [hunk 4]
```diff
@@ -39,54 +39,61 @@ You can find more information about the Polkadot staking dashboard on the dedica
...
-  <td>
-    <a class="guide-link" href="https://youtu.be/dDIG7QAApig?feature=shared">  
-      <img class="guide-image" src="https://img.youtube.com/vi/dDIG7QAApig/0.jpg" alt="Drawing" width="250" height="150"/>
-              <div class="cards-body">
-                  <h5 class="cards-title">Join a Nomination Pool</h5>
-              </div>
+    <p>
+      Nominate
+      <br />
+      <a href="https://youtu.be/F59N3YKYCRs?feature=shared">» view video</a>
```
Comment (profradha) [inline docs/general/staking-apps.md]:
```suggestion
      <a href="https://youtu.be/F59N3YKYCRs?feature=shared">Stake your DOT</a>
```

**Review by DrW3RK — APPROVED** [review]
Looks good. Just modify the links like below and the PR is good to go. Thank you.

**Pull request merged**